src/gaen/nativeModule.ts

```diff
@@ -50,6 +50,34 @@ export const subscribeToEnabledStatusEvents = (
   )
 }
 
+export const subscribeToBluetoothStatusEvents = (
+  cb: (enabled: boolean) => void,
+): EventSubscription => {
+  const ExposureEvents = new NativeEventEmitter(
+    NativeModules.ExposureEventEmitter,
+  )
+  return ExposureEvents.addListener(
+    "onBluetoothStatusUpdated",
```

Comment:
Curious if this event is also registered on the iOS side? If it isn't should we guard against it?

Reply:
We are listening to that event on both platforms.
iOS is not emitting it, but that shouldn't be an issue